### PR TITLE
feat(styles): use GitHub syntax theme

### PR DESCRIPTION
**Pull request by cmoog**

### CODE REVIEW

File: public/styles/hljs/default.min.css [hunk 1]
```diff
@@ -0,0 +1,9 @@
+/*!
```
Comment (lucacasonato) [inline public/styles/hljs/default.min.css:1]: The content in this file seems to be duplicated with the content in the GitHub css files.

Comment (cmoog) [reply public/styles/hljs/default.min.css]: Good catch. Just removed the duplicate.

**Comment by kt3k** [comment]
Does this address any issue?

**Comment by cmoog** [comment]
There isn't an issue yet. I'll just close this out and let others push through theme changes if there is enough will.

**Pull request closed without merging**